Source/WTF/wtf/SegmentedVector.h

```diff
@@ -29,11 +29,10 @@
 #pragma once
 
 #include <wtf/MallocCommon.h>
+#include <wtf/MallocPtr.h>
 #include <wtf/Noncopyable.h>
```

Comment:
Pretty sure we don’t need to include `MallocCommon.h` any more and we definitely don’t need `Noncopyable.h` since `MallocPtr.h` already includes that.
```suggestion
#include <wtf/MallocPtr.h>
```